src/aura/AuraAvatarMultiToken.sol

```diff
@@ -574,94 +574,60 @@ contract AuraAvatarTwoToken is BaseAvatar, PausableUpgradeable, AuraAvatarUtils,
 
-            emit Withdraw(address(asset1), _amountAsset1, block.timestamp);
-        }
-        if (_amountAsset2 > 0) {
-            baseRewardPool2.withdrawAndUnwrap(_amountAsset2, false);
-            asset2.safeTransfer(msg.sender, _amountAsset2);
+            IBaseRewardPool(crvRewards).withdrawAndUnwrap(_bptsDeposited[i], false);
+            IERC20MetadataUpgradeable(lpToken).safeTransfer(msg.sender, _bptsDeposited[i]);
 
-            emit Withdraw(address(asset2), _amountAsset2, block.timestamp);
+            emit Withdraw(lpToken, _bptsDeposited[i], block.timestamp);
         }
     }
 
     /// @notice A function to process pending BAL and AURA rewards at regular intervals.
-    /// @param _auraPriceInUsd A reference price for AURA in USD (in 8 decimal precision) to compare with TWAP price. Leave as 0 to use only TWAP.
-    function _processRewardsKeeper(uint256 _auraPriceInUsd) internal {
+    function _processRewardsKeeper() internal {
         uint256 lastClaimTimestampCached = lastClaimTimestamp;
         uint256 claimFrequencyCached = claimFrequency;
         if ((block.timestamp - lastClaimTimestampCached) < claimFrequencyCached) {
             revert TooSoon(block.timestamp, lastClaimTimestampCached, claimFrequencyCached);
         }
 
-        _processRewards(_auraPriceInUsd);
+        _processRewards();
     }
 
     /// @notice Claim and process BAL and AURA rewards, selling some of it to USDC and depositing the rest to bauraBAL
     ///         and vlAURA.
-    /// @param _auraPriceInUsd A reference price for AURA in USD (in 8 decimal precision) to compare with TWAP price. Leave as 0 to use only TWAP.
     /// @return processed_ An array containing addresses and amounts of harvested tokens (i.e. tokens that have finally
     ///                    been swapped into).
-    function _processRewards(uint256 _auraPriceInUsd) internal returns (TokenAmount[] memory processed_) {
+    function _processRewards() internal returns (TokenAmount[] memory processed_) {
         // 1. Claim BAL and AURA rewards
         (uint256 totalBal, uint256 totalAura) = claimAndRegisterRewards();
 
-        // 2. Swap some for USDC and send to owner
-        uint256 balForUsdc = (totalBal * sellBpsBalToUsdc) / MAX_BPS;
-        uint256 auraForUsdc = (totalAura * sellBpsAuraToUsdc) / MAX_BPS;
-
-        uint256 usdcEarnedFromBal;
-        if (balForUsdc > 0) {
-            usdcEarnedFromBal = swapBalForUsdc(balForUsdc);
+        uint256 totalUsdcEarned;
+        if (totalBal > 0) {
+            totalUsdcEarned = swapBalForUsdc(totalBal);
         }
-        uint256 usdcEarnedFromAura;
-        if (auraForUsdc > 0) {
-            usdcEarnedFromAura = swapAuraForUsdc(auraForUsdc, _auraPriceInUsd);
-        }
-
-        uint256 totalUsdcEarned = usdcEarnedFromBal + usdcEarnedFromAura;
 
         address ownerCached = owner();
         if (totalUsdcEarned > 0) {
             USDC.safeTransfer(ownerCached, totalUsdcEarned);
```

Comment:
Guess we don't need to cache `owner` anymore

Reply:
removed in 1bf3520289ab8590692162543359a3039d54602b